apps/src/weblab/CdoBramble.js

```diff
@@ -745,6 +762,10 @@ export default class CdoBramble {
     this.brambleProxy?.refreshPreview();
   }
 
+  fileRefresh(callback = () => {}) {
```

Comment:
note about this change: i forgot to include this method in #39831, so i've added it here.